CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+
+## [unreleased]
+- Adds APIs for managing tenants
+  - adds `/recipe/multitenancy/tenant` PUT
+  - adds `/recipe/multitenancy/tenant/remove` POST
+  - adds `/recipe/multitenancy/tenant/config` GET
```

Comment:
missed updatying change log for listing tenants API...

Reply:
Done